mobile/assets/i18n/en.json

```diff
@@ -1,6 +1,16 @@
 {
     "newBooks": "New Books for You",
     "allBooks": "All Books",
+    "letsTalk": "Let's Talk",
+    "tel": "Tel: ",
+    "num": "858.274.9673",
+    "fax": "Fax: ",
+    "days": "Mon-Fri: ",
+    "hours": "8am-4pm",
+    "emailUs": "Email Us: ",
+    "emailAddress": "info@wordsalive.gov",
+    "addressTitle": "Address: ",
+    "actualAddress": "Santa Fe Street Suite 219 San Diego, California, 92081, United States",
```

Comment:
I feel like the names and values of these translations could be refactored a little more clearly.

First of all, stuff like punctuation doesn't need to be translated, so I think we can move the colons here outside of i18n.

I also don't think the address or email address will need to be translated - they're really only valid in English. Phone number arguably _maybe_ could be translated so that we could include the area code internationally, but langauge != location, so maybe we should just add the area code to the number in all languages. Lmk your thoughts here.

Lastly, i18n translation keys are usually just the English value in camel case, which makes the code easier to read. I think if you remove the num, emailAddress, and actualAddress keys that should be a pretty easy change.